File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.py
def solution(a, b):
    
    answer = 0
    
    if(a > b) :
        n = b
        m = a
    if(a == b) :
        n = a
        m = a
    if(a < b) :
        n = a
        m = b
        
    for i in range(n, m+1) :
        answer += i
    return answer

File: 프로그래머스/1/12912. 두 정수 사이의 합/두 정수 사이의 합.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        int n = 0;
        int m = 0;
        if(a <= b) {
            n = a;
            m = b;
        }
        else {
            n = b;
            m = a;
        }
        for(int i = n; i < m + 1; i++){
            answer += i;
        }
        return answer;
    }
}